.github/workflows/wave.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     timeout-minutes: 60
     strategy:
       fail-fast: false
-      max-parallel: 4
+      max-parallel: 2
```

Comment:
why not 4?

Reply:
Why set a limit at all? Idk, I think it was to not soak up a ton of runners.